linchpin/shell/__init__.py

```diff
@@ -336,8 +340,23 @@ def fetch(ctx, fetch_type, remote, root):
               metavar='TX_ID', required=False,
               help="Display a specific transaction by ID (tx_id)."
                    " Only works with '--view=tx'")
+@click.option('--output-format', type=str, default="cfg",
+              metavar='output_format', required=False,
+              help="Inventory output format")
```

Comment:
What is the difference between this and `--inventory-format` in the `up()` method? Seems these options should be combined and put into a general option. I'd go with `--output-format`. 

The one challenge with this is that there is already a `--output-pinfile` general option, and the standard output for that is 'json'. It seems to me that this would be a good time to move the default of `--output-format` to 'json'. While this might not be backward compatible in terms of format, it should be in terms of functionality.